### ISSUE TITLE
`require-computed-property-dependencies` rule does not detect complete missing dependency when using a combination of both ES5 getters and `get`

### ISSUE THREAD

**Issue by bmish** [ember-cli/eslint-plugin-ember#897]
The [rule](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/require-computed-property-dependencies.md) only detects `hello` as the missing dependency in this example:

```js
myProp: computed(function () {
  return this.hello.get('world');
})
```

So it gets autofixed to:

```js
myProp: computed('hello', function () {
  return this.hello.get('world');
}),
```

However, this is an edge case for multiple reasons, including that we have the recommended [no-get](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-get.md) rule to prevent `get` usage, so I will close this issue
